src/README.md

```diff
@@ -0,0 +1,60 @@
+# DMD's source code
+
+This directory contains the source code of the dmd compiler.
+Refer to [this file](dmd/README.md) for an overview of the actual source files.
+
+## Building the compiler
+
+The compiler can be built using the integrated build system `build.d`.
+
+
+```console
+./build.d
+```
```

Comment:
List the requirements for this to work - what version of dmd to be installed and on the path.

Reply:
Done